superbench/runner/ansible.py

```diff
@@ -29,13 +29,16 @@ def __init__(self, config):
         }
         if config:
             inventory_file = getattr(config, 'host_file', None)
-            if inventory_file:
-                self._config['inventory'] = inventory_file
+            inventory_list = getattr(config, 'host_list', '').strip(',')
+            if inventory_file or inventory_list:
+                self._config['inventory'] = inventory_file or inventory_list
                 self._config['host_pattern'] = 'all'
-                inventory = InventoryManager(loader=DataLoader(), sources=inventory_file)
+                inventory = InventoryManager(loader=DataLoader(), sources=inventory_file or f'{inventory_list},')
                 host_list = inventory.get_groups_dict()['all']
                 if len(host_list) > 0:
                     self._config['cmdline'] = '--forks {}'.format(len(host_list))
+                if inventory_list in ['localhost', '127.0.0.1']:
```

Comment:
what will happen if the inventory_list is 'localhost,node2,node3'?

Reply:
it will try to connect to three nodes using ssh, for fine-grained config (local connect to localhost, ssh to others), an inventory file is needed

Reply:
Do we have comments or description for these kind of cases?

Reply:
nope, the `--host-list` is mainly targeted on localhost test, so you don't need to prepare a config file.
it's recommended to prepare a host config for other cases, so we only document how to prepare config files for those cases, including [mixing localhost and remote](https://microsoft.github.io/superbenchmark/docs/getting-started/configuration#ansible-inventory)